### PR TITLE
Add support for StarFive device

### PR DESCRIPTION
**Pull request by Errant404**
Add support for StarFive device to aosc-mkrawimg.
Currently only jh7110-starfive-visionfive-2 is supported.

### CODE REVIEW

File: solutions/starfive-base [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+			/dev/mmcblk1p3 /boot vfat  defaults               0 2
+			/dev/mmcblk1p4 /     btrfs defaults,compress=zstd 0 1
+		EOF
+
+		echo "Configuring U-Boot ..."
+		cat > etc/default/u-boot <<- EOF
+			U_BOOT_UPDATE="true"
+			U_BOOT_PARAMETERS="rw console=tty0 console=ttyS0,115200 earlycon rootwait stmmaceth=chain_mode:1 selinux=0"
+			U_BOOT_ROOT="root=/dev/mmcblk1p4"
+			U_BOOT_FDT_DIR="/dtbs/dtbs-"
```
Comment (Icenowy) [inline solutions/starfive-base:62]: `dtbs-` ?

Comment (Errant404) [reply solutions/starfive-base:62]: This is the prefix of the directory where the device tree files are located, followed by the VERSION of the Kernel.
Here is the [relevant source code](https://salsa.debian.org/debian/u-boot-menu/-/blob/debian/master/u-boot-update?ref_type=heads#L144).

**Comment by Icenowy** [comment]
```
Enabling presets ...
Failed to enable unit: Unit nvidia-persistenced.service does not exist
Enabling DKMS ...
Failed to enable unit: Unit dkms.service does not exist
Creating a default user ...
Setting default password ...
Enabling RootFS adjustment service ...
Failed to enable unit: Unit rootfs-grow.service does not exist
```

Error is present when building the image ...

**Comment by Errant404** [comment]
> ```
> Enabling presets ...
> Failed to enable unit: Unit nvidia-persistenced.service does not exist
> Enabling DKMS ...
> Failed to enable unit: Unit dkms.service does not exist
> Creating a default user ...
> Setting default password ...
> Enabling RootFS adjustment service ...
> Failed to enable unit: Unit rootfs-grow.service does not exist
> ```
> 
> Error is present when building the image ...

These errors are expected, and they should not affect the generation of the image.

**Review by Icenowy — APPROVED** [review]
Looks like it boots now.

LGTM.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**